docs/source/index.rst

```diff
@@ -147,4 +147,6 @@ More Resources
    :maxdepth: 2
    :hidden:
 
-   /APIs/index.rst
+   /APIs/data_preparation.rst
+   /APIs/conversation_memory.rst
+   /APIs/transforms.rst
```

Comment:
These look a little strange b/c you get the little triangle dropdown, but since they don't have any subpages, it doesn't do anything when you click on them. The links work if you click on the text, but it's a bit confusing. Let me see if there is a way to not render the dropdowns.